progress/ansimeter.go

```diff
@@ -152,11 +195,11 @@ func (p *ANSIMeter) Set(current float64) {
 	rtimeleft := []rune(timeleft)
 	msg := make([]rune, 0, col)
 	// XXX: assuming terminal can display `col` number of runes
-	msg = append(msg, norm(col-len(rpercent)-len(rspeed)-len(rtimeleft), p.label)...)
+	msg = append(msg, norm(col-runeWidth(rpercent)-runeWidth(rspeed)-runeWidth(rtimeleft), p.label)...)
```

Comment:
Is this needed? `rpercent`, `rspeed` and `rtimeleft` will be runes of size 1 any way so it enough to use `len`, no?

Reply:
Thanks for your review.

You are right, we don't need apply runeWidth on `rpercent` and `rspeed`.

But for `rtimeleft`, it is a localized text using `quantity.FormatDuration`. Personally, I don't think these need to be translated. After all, 1m30s is not completely English, and CJK speakers understand it to mean 1 minute and 30 seconds. Also, the current translation is incorrect. In zh_CN locales, 1m30s is translated as `1月30秒`, which means 1 month and 30 seconds, not 1 minute and 30 seconds.

However, this patch does not intend to solve the translation problem. Let me fix `rpercent` and `rspeed`, and keep `rtimeleft`, so that the display is normal.